src/java_bytecode/java_utils.cpp

```diff
@@ -182,3 +184,16 @@ dereference_exprt checked_dereference(const exprt &expr, const typet &type)
   result.set(ID_java_member_access, true);
   return result;
 }
+
+void java_add_components_to_class(
```

Comment:
Shouldn't this should be declared above? ~I'm not sure how you can call it without a declaration~ Don't declare in a different header file

Reply:
Also documentation.